services/addons/tests/geo_msg_query/test_geo_query.py

```diff
@@ -40,6 +40,33 @@ def test_create_message_bsm():
     assert create_message(original_message, msg_type) == expected_message
 
 
+def test_create_message_bsm_millis():
```

Comment:
Since we're specifying the units in this test title, I think units should be specified in the `test_create_message_bsm` test above as well.